.github/workflows/release.yaml

```diff
@@ -163,6 +163,14 @@ jobs:
           echo "${TARGET_VERSION}" > VERSION
           git commit -m "Bump version to ${TARGET_VERSION}" VERSION
 
+      - name: Update README.md link
+        run: |
+          set -ue
+          echo "Bumping Integration tests link"
+          export BASE_VERSION=$(echo ${TARGET_VERSION} | perl -pne 's/^(\d+\.\d+).*/$1/')
+          perl -pi -e 'next if $done; next unless m{Integration.*tests/badge\.svg}; s{(branch(?:=|%3A))[0-9a-zA-Z-.]+}{$1release-$ENV{BASE_VERSION}}g; $done=1' README.md
+          git diff
+          git commit README.md -m "Update
```

Comment:
While I made friends with the idea to have URLs pointing to their release's branch, I really think this change does not belong in the release action (which assumes the release branch is already existing). Basically, implementing it here would be a no-op for every release within a branch except the first. And also, it's hard to test here.

Just an idea, maybe we should externalize this, and provide a new script for creating the release branch. AFAIK, it's done manually right now. The more automation we have for such things, the better. We could put a lot more stuff than updating the link in the README from such a script, and it would also be a lot easier to test & maintain. What do you think?

Reply:
To the current proposed change, you're right that it should be a no-op for future steps along a branch, but the time cost today would be minimal (and it shouldn't result in a commit entry), so it isn't a big problem.

Your proposal sounds reasonable in principle. But, I think that's more work than I'm up for now. It's easy for me to shoehorn a small change into an existing framework. It's much riskier for me to try to migrate an amorphous task sequence (that I've never performed -- and can't perform & validate) into a script. -- I also don't think it makes sense for me to file the issue since I really don't understand it well enough / couldn't even validate/verify it on the other end. -- OTOH, i'd be happy to 👍  it were you to file it 😄 .

If you like, I can just drop this change and we can merge w/o it, or you could merge w/ this change and then once you've created that script migrate this code to it.